src/client/src/rt-components/platform/adapters/openfin/openFin.ts

```diff
@@ -4,6 +4,15 @@ import { openDesktopWindow } from './window'
 import { fromEventPattern } from 'rxjs'
 import { excelAdapter } from './excel'
 import { CurrencyPairPositionWithPrice } from 'rt-types'
+import { LayoutActions } from '../../../../shell/layouts/layoutActions'
+
+export async function setupWorkspaces(store: any) {
+  if (typeof fin !== 'undefined') {
+    const workspaces = require('openfin-layouts').workspaces
```

Comment:
Any need for inline require?

Reply:
I had issue at first with the browser... but I tried again and it looks good so done, I removed the inline require.